src/test/unit_tests/test_s3_bucket_policy.js

```diff
@@ -168,25 +165,26 @@ mocha.describe('s3_bucket_policy', function() {
         }).promise());
     });
 
-    mocha.it('should be able to set bucket policy when none set', async function() {
-        const policy = {
-            Version: '2012-10-17',
-            Statement: [{
-                Sid: 'id-1',
-                Effect: 'Allow',
-                Principal: { AWS: user_a },
-                Action: ['s3:GetBucketPolicy'],
-                Resource: [`arn:aws:s3:::${BKT}`]
-            }]
-        };
-        await s3_owner.deleteBucketPolicy({ // should work - owner can always delete the buckets policy
-            Bucket: BKT,
-        }).promise();
-        await s3_a.putBucketPolicy({ // user a have FC to bucket
-            Bucket: BKT,
-            Policy: JSON.stringify(policy)
-        }).promise();
-    });
+    // @utkarsh-pro: Should no longer work
+    // mocha.it('should be able to set bucket policy when none set', async function() {
```

Comment:
maybe change to owner and not any user. but in aws in a bucket with no bucket policy can't any user set one - that's what I remember - but please validate ( by testing in AWS console)